components/table/src/Table.js

```diff
@@ -20,15 +20,15 @@ const TableWrapper = ({ bordered, responsive, withinPanel, ...props }) => {
 	return responsive ? (
 		<div
 			css={[
-				withinPanel && { border: 0, marginBottom: 0 },
+				extraStyles,
 				{
 					[xsOnly]: {
 						width: '100%',
-						marginBottom: withinPanel ? 0 : '1.125rem',
+						marginBottom: '1.125rem',
 						overflowY: 'hidden',
 						overflowX: 'auto',
 						// -ms-overflow-style: '-ms-autohiding-scrollbar',
-						border: withinPanel ? 0 : `1px solid ${COLORS.border}`,
+						border: `1px solid ${COLORS.border}`,
 						// -webkit-overflow-scrolling: 'touch',
```

Comment:
Need this?

Reply:
see https://github.com/WestpacGEL/GEL/pull/157#discussion_r330805089

Reply:
referring to commented out `// -webkit-overflow-scrolling: 'touch',`

Reply:
removed! 👍